tests/integration/store/test_component_props.py

```diff
@@ -0,0 +1,69 @@
+import json
+import time
+from pytest import approx
+import dash
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+def test_stcp100_clear_data_on_all_types(store_app, dash_duo):
```

Comment:
the three digit here plays another role as sub category,  1xx will be test on `clear_data` property
and 2xx will on `modified_timestamps`